src/app.ts

```diff
@@ -315,13 +315,20 @@ function supportsFancyCharacters(): boolean {
   // There is no sure-fire way, but we know that most problems are when running in the IG Publisher,
   // so try to detect that situation (which is still actually pretty tricky and not guaranteed).
 
-  // 1. Many JVM will insert an environment variable indicating the main Java class being run.
+  // 1. If the user wants the fancy characters, allow them
+  if (
+    Object.keys(process.env).some(k => k === 'FORCE_FANCY_CHARACTERS') &&
+    !!process.env.FORCE_FANCY_CHARACTERS
```

Comment:
I don't think this quite works as intended, as `!!'false'` is `true` (because a non-empty string is truthy in JS). Perhaps you should use `/(true|1)/i.test(process.env.FORCE_FANCY_CHARACTERS)` to do a case-insensitive test against true and 1.

Reply:
Oh yeah, of course, environment variables are only strings. Sorry :)